addons/website_blog/data/website_blog_demo.xml

```diff
@@ -329,5 +336,68 @@ Naturally, as you grow in your love of astronomy, you will find many celestial b
 ]]>
             </field>
         </record>
+
+        <record id="blog_comment_1" model="mail.message">
+            <field name="body">Beautiful !, I know where will be my next trip !</field>
+            <field name="model">blog.post</field>
+            <field name="res_id" ref="blog_post_1"/>
+            <field name="message_type">comment</field>
+            <field name="author_id" ref="base.partner_admin"/>
+            <field name="subtype_id" ref="mail.mt_comment"/>
+        </record>
+
+        <record id="blog_comment_2" model="mail.message">
+            <field name="body">Hi ! How much time did you went there ? </field>
+            <field name="model">blog.post</field>
+            <field name="res_id" ref="blog_post_1"/>
+            <field name="message_type">comment</field>
+            <field name="author_id" ref="base.partner_demo"/>
+            <field name="subtype_id" ref="mail.mt_comment"/>
+        </record>
+
+        <record id="blog_comment_3" model="mail.message">
+            <field name="body">I'll follow your instructions next time ! It will save myself from weird place like last time :D</field>
```

Comment:
Did someone gave you the sentences to use or did you make them yourself? This one is really weird for a weird mind like mine. Also there are some typo in those comments.

cc @sbuhl

Reply:
I did them myself as @sbuhl asked me. But feel free to adjust it, wasn't that much inspired haha. I'll change this comment

Reply:
"I'll follow your instructions next time! It will save myself from a weird place like last time :D"
@qsm-odoo I found this one quite funny

Reply:
Haha. So what should I do with this one ?

Reply:
You can use @sbuhl 's corrected version